README.md

```diff
@@ -311,6 +311,8 @@ jobs:
   id: coverage_comment
   uses: py-cov-action/python-coverage-comment-action@v3
   with:
+    GITHUB_BASE_URL: https://api.github.com
+
     GITHUB_TOKEN: ${{ github.token }}
 
```

Comment:
```suggestion
    GITHUB_TOKEN: ${{ github.token }}
    
    # Change this in case you use GitHub Entreprise with a different API endpoint
    GITHUB_BASE_URL: https://api.github.com

```